File: test/go/exchangeInfo.go
package main

import (
	"fmt"

	"mexc-sdk/mexcsdk"
)

func main() {
	apiKey := "your apikey"
	apiSecret := "your apiSecret"
	client := mexcsdk.NewSpot(&apiKey, &apiSecret)

	options := map[string]interface{}{"symbol": "BTCUSDT"}
	info := client.ExchangeInfo(options)
	fmt.Println(info)
}

File: test/js/exchangeInfo.js
const Mexc = require('../../dist/js/package')

const client = new Mexc.Spot()
client.config.apiKey = "your apiKey"
client.config.apiSecret = "your apiSecret"

const singleRes = client.exchangeInfo({ symbol: "BTCUSDT" })
console.log(singleRes)